reporting/test/helpers/scenarios.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WhoTracks.Me
+ * https://ghostery.com/whotracksme
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { brotliDecompressSync } from 'zlib';
+
+async function download(release, scenarior, browser) {
+  const scenartioFileName = `events_${scenarior}_${browser}.log`;
+
+  const scenariosPath = path.join(
+    import.meta.dirname,
+    '..',
+    '..',
+    'scenarios',
+    release,
+  );
+  if (!fs.existsSync(scenariosPath)) {
+    fs.mkdirSync(scenariosPath, { recursive: true });
+  }
+
+  const scenariorPath = path.join(scenariosPath, scenartioFileName);
```

Comment:
Can you please do some global renamings like:
scenarior -> scenario
scenartio -> scenario

Reply:
done